.coderabbit.yaml

```
@@ -0,0 +1,163 @@
+  collapse_walkthrough: false
+  
+# Path filters - exclude files from analysis
+path_filters:
+  # Exclude directories that aren't critical for code review
+  exclude:
+    # Dependencies and package files
+    - "node_modules/**"
+    - "package-lock.json"
+    - "yarn.lock"
+    - "pnpm-lock.yaml"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)